### DISCUSSION TITLE
Why the logs signal is frozen for this project?

### DISCUSSION THREAD

**Discussion by PalanQu** [open-telemetry/opentelemetry-go#3472] [General]
### Problem Statement

I think log is the one of important part in opentelemetry, why the deployment is halted? Any help can I do for this?

**Comment by dmathieu** [comment]
Logs are frozen for two main reasons:
To allow us to move forward on metrics GA.
And because their API is not finalized in the specifications themselves: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/api.md

I think the best way to help logs move forward is to join the `#otel-logs` CNCF slack channel, as well as that SIG's meeting and get involved there.

**Reply by PalanQu** [reply]
Thanks for your reply!
